components/homme/src/interpolate_driver_mod.F90

```diff
@@ -540,9 +540,9 @@ subroutine interpolate_vars(infile, outfile, elem, par, interpdata)
     real(kind=real_kind), pointer :: zeta(:,:,:,:),div(:,:,:,:)
 
     integer(kind=nfsizekind) :: start(3), count(3)
-    integer :: starti(3), counti(3)
+    integer :: starti(3), counti(3), index(1)
```

Comment:
How did this bug come up?

Reply:
here's some more info:  https://github.com/E3SM-Project/scorpio/issues/327

it's probably been in there for a long time, since most people stopped using HOMME's built in interpolation tool several years ago